DNN Platform/Library/Entities/Users/Profile/UserProfile.cs

```diff
@@ -61,9 +61,9 @@ public class UserProfile : IIndexable
         public const string USERPROFILE_Unit = "Unit";
         public const string USERPROFILE_Street = "Street";
         public const string USERPROFILE_City = "City";
-        public const string USERPROFILE_Region = "Region";
         public const string USERPROFILE_Country = "Country";
-        public const string USERPROFILE_PostalCode = "PostalCode";
+        public const string USERPROFILE_Region = "Region";
+	public const string USERPROFILE_PostalCode = "PostalCode";
```

Comment:
Can you just adjust the spacing here, other than that, I am fine with this. For other reviewers this does not change behaviour just the order of declarations so it matches in code what we have in the UI for most situations.